styles.css

```diff
@@ -0,0 +1,89 @@
+
+.images {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+  position: relative;
+  margin-top: 40px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.time-wifi-battery {
+  width: 10px;
+  height: 12px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.wifi-battery {
+  float: right;
+  margin-right: 20px;
+}
+
+#headline {
+  width: 375px;
+  height: 700px;
+  margin: 0 auto;
+  background-image: url(./images/header-shapes\ mobile@2x.svg);
+  border-bottom-left-radius: 80px;
+}
+
+body.active {
+  overflow-y: hidden;
+}
+
+.header {
+  background-image: url();
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: var(--header-color);
+  padding: 128px 24px 0;
+}
```

Comment:
Great design @Khadaro 🤝

- [ ] Just like the previous reviewer pointed out, kindly make your design a replica of the Figma template provided.
- Your design
![your design](https://user-images.githubusercontent.com/63917725/185675450-d473918a-683c-4075-a40c-01381cbe8ade.PNG)
- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/185675713-0fd7945d-d7b1-4b3c-966b-4d5c8511e255.PNG)